core-services/release-controller/_releases/release-ocp-4.4-ci.json

```diff
@@ -24,7 +24,6 @@
           "libvirt-installer",
           "baremetal-machine-controllers",
           "baremetal-operator",
-          "baremetal-runtimecfg",
```

Comment:
Is this changed in the other branches (specifically 4.5)?

Reply:
Updated 4.5 and 4.3